pstress/pstress-run-PXC57.conf

```diff
@@ -92,7 +92,7 @@ PXC_MAX_NR_OF_RND_OPTS_TO_ADD=4
 # Save only trials that generate a core file (good for initial few runs where  #
 # there are lot of crashes/asserts)                                            #
 ################################################################################
-SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=0
+SAVE_TRIALS_WITH_CORE=0
 
 ################################################################################
 # Saves per-trial SQL even if SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=1.        #
```

Comment:
Fix the comment message in all the conf files.
use: grep -inr "valgrind" * to find out all the places where valgrind is used and fix it.